Floofbot/Modules/Administration.cs

```diff
@@ -165,6 +165,193 @@ public async Task warnlog([Summary("user")] string user)
         }
 
 
+        [Command("mute")]
+        [Summary("Applys a mute role to a user")]
+        [RequireContext(ContextType.Guild)]
+        [RequireUserPermission(GuildPermission.ManageMessages)]
+        public async Task MuteUser([Summary("user")]string user, [Summary("Time")]string time = null)
+        {
+            IUser badUser = resolveUser(user);
+            if (badUser == null) {
+                await Context.Channel.SendMessageAsync($"⚠️ Could not find user \"{user}\"");
+                return;
+            }
+
+            IRole mute_role;
+
+            //check to see if the server exists within the "AdminConfig" Table
+            if (!_floofDB.AdminConfig.AsQueryable().Any(x => x.ServerId == Context.Guild.Id)) {
+
+                //create new mute role
+                mute_role = await createMuteRole();
+
+                //save the newly created role
+                _floofDB.Add(new AdminConfig {
+                    ServerId = Context.Guild.Id,
+                    MuteRoleId = mute_role.Id
+                });
+                _floofDB.SaveChanges();
+            }
+            else {
+                //grabs the mute role from the database
+                mute_role = Context.Guild.GetRole(
+                   _floofDB.AdminConfig.AsQueryable()
+                   .Where(x => x.ServerId == Context.Guild.Id)
+                   .Select(x => x.MuteRoleId).ToList()[0]);
+
+                //mute role was deleted create a new one
+                if (mute_role == null) {
+                    mute_role = await createMuteRole();
+                    var result = _floofDB.AdminConfig.AsQueryable()
+                         .SingleOrDefault(x => x.ServerId == Context.Guild.Id);
+                    result.MuteRoleId = mute_role.Id;
+                    _floofDB.SaveChanges();
+                }
+            }
+
+            if (Context.Guild.GetUser(badUser.Id).Roles.Contains(mute_role)) {
+                await Context.Channel.SendMessageAsync($"{badUser.Username}#{badUser.Discriminator} is already muted!");
+                return;
+            }
+
+                await Context.Guild.GetUser(badUser.Id).AddRoleAsync(mute_role);
```

Comment:
Fix indent here. Actually, you can probably apply the formatting keyboard shortcut to get the whole file formatted.